python/jupytergis_lab/jupytergis_lab/explore.py

```diff
@@ -0,0 +1,75 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Literal, Optional
+
+from jupytergis_lab import GISDocument
```

Comment:
```suggestion
from . import GISDocument
```

Reply:
I'm not sure about this, but it could be my ignorance; when I'm writing a Python package, I always import from the package name explicitly, no relative imports. Why should I prefer importing from `.`?

Reply:
If this is holding up merge, please feel free to commit it!